src/source-updater.js

```diff
@@ -139,4 +133,10 @@ export default class SourceUpdater {
       callbacks[0]();
     }
   }
+
+  dispose() {
+    if (this.sourceBuffer_ && this.mediaSource.readyState === 'open') {
+      this.sourceBuffer_.abort();
```

Comment:
In this case no one should be creating a source updater without passing in a media source (I am pretty sure).